sonar-plugin/bridge/src/main/java/org/sonar/plugins/javascript/bridge/FormDataUtils.java

```diff
@@ -85,11 +97,12 @@ public static BridgeServer.BridgeResponse parseFormData(HttpResponse<byte[]> res
   @CheckForNull
   private static Node parseProtobuf(byte[] ast) throws IOException {
     try {
-      return Node.parseFrom(ast);
+      CodedInputStream input = CodedInputStream.newInstance(ast, 0, ast.length);
```

Comment:
_nitpick_ the api proposes a simpler call, wdyt?
```suggestion
      CodedInputStream input = CodedInputStream.newInstance(ast);
```